core/iwasm/common/wasm_memory.c

```diff
@@ -147,23 +147,70 @@ wasm_runtime_free_internal(void *ptr)
 void *
 wasm_runtime_malloc(unsigned int size)
 {
+#if 0
     if (size == 0) {
         LOG_WARNING("warning: wasm_runtime_malloc with size zero\n");
         /* At lease alloc 1 byte to avoid malloc failed */
         size = 1;
     }
 
     return wasm_runtime_malloc_internal(size);
+#endif
+    return wasm_gc_malloc(size);
 }
 
 void *
 wasm_runtime_realloc(void *ptr, unsigned int size)
 {
+#if 0
     return wasm_runtime_realloc_internal(ptr, size);
+#endif
```

Comment:
Same as above, normally wasm_runtime_realloc should work and is required by loader and wasm_enlarge_memory.